pytorch_lightning/plugins/precision/deepspeed_precision.py

```diff
@@ -54,7 +55,13 @@ def backward(
 
         return closure_loss
 
-    def clip_gradients(self, optimizer: Optimizer, clip_val: Union[int, float], norm_type: float = float(2.0)):
+    def clip_gradients(
+        self,
+        optimizer: Optimizer,
+        clip_val: Union[int, float],
+        gradient_clip_algorithm: str = GradClipAlgorithmType.NORM,
+        norm_type: float = float(2.0),
```

Comment:
is the float needed?
```suggestion
        norm_type: float = 2.0,
```